src/commands/profiles/create.js

```diff
@@ -230,7 +241,7 @@ class ProfilesCreate extends BaseCommand {
       this.logger.debug(error);
       throw new TwilioCliError('Could not create an API Key.');
     }
-
+    await this.removeKeytarKeysByProfileId(this.profileId);
```

Comment:
To confirm, we don't fail even when we cannot remove from keytar, right? We just proceed with removing projects entry and creating the new profiles entry. May need a unit test here

Reply:
[Yes](https://github.com/twilio/twilio-cli/pull/264/files#diff-b6e7e69a3c909f6ead791bd2fb184d2e7c0b08d6df3b92397f6e271e434be35dR227), just logging as warning and proceeding with removing the entry in projects. As we moved to creating profile in config file, I need to find a way to test it against existing profile that is in Projects and Keytar (There are no existing test cases for updating profiles currently).

Reply:
Added the test cases accordingly.

Reply:
Is `this.profileId` sanitized at this point? If not, we might want to sanitize here